MonoGame.Framework/Platform/Graphics/RenderTarget2D.DirectX.cs

```diff
@@ -143,38 +138,57 @@ internal void ResolveSubresource()
             lock (GraphicsDevice._d3dContext)
             {
                 GraphicsDevice._d3dContext.ResolveSubresource(
-                    _msTexture,
+                    GetMSTexture(),
```

Comment:
We use GetMSTexture(),  GetTexture(), instead of the members to ensure creation of the textures.